docs/extending/admin_views.md

```diff
@@ -102,8 +97,21 @@ Now create a `templates/wagtailcalendar/` folder within the `wagtailcalendar` ap
 {% endblock %}
 ```
 
+```css
+/* calender.css */
+table.month {
+            margin: 20px;
+        }
+        table.month td, table.month th {
+            padding: 5px;
+        }
+
```

Comment:
Just watch the formatting here, it's a bit out of wack. We don't run prettier on our docs but I think this can be a bit cleaner.

Reply:
Done, all left aligned with proper indentation. I'll commit the changes